algorithms/ar-msrt/javascript/merge_sort.js

```diff
@@ -0,0 +1,36 @@
+function mergeSort(array) {
+    const merge = function (array1, array2) {
+        if (array1 == [] || array1 == undefined) {
+            return array2;
+        }
+        else if (array2 == [] || array2 == undefined) {
```

Comment:
Codacy found an issue: [Unexpected use of undefined.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6334421)